src/components/GraphingSystemsOfLinearInequalities.js

```diff
@@ -60,32 +59,28 @@ class GraphingSystemsOfLinearInequalities extends React.Component {
                     <img 
                         src="https://i.ibb.co/Qfc9QB2/Inequalities-A.png"
                         alt="Answer Choice A"
-                        width={200}
-                        height={200} />
-                    <p className="p-body-small">
+                        style={{ width: "18%" }} />
```

Comment:
I don't think it's necessary to make the images in the answer choices responsive as they are already relatively small, and on a mobile device, they are unreadable.